src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,39 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_NUM_OF_ITEMS = 10;
+    private K[] keys = (K[]) new Object[MAX_NUM_OF_ITEMS];
+    private V[] values = (V[]) new Object[MAX_NUM_OF_ITEMS];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
class variables have default values